app/views/controllers/projects/violations/edit.html.erb

```diff
@@ -0,0 +1,31 @@
+<%
+  add_page_title(:violations_edit_title.t)
+  @container = :wide
+%>
+
+<h4 class="pb-3">
+  <%= :PROJECT.l + ": " + link_to_object(@project).t %>
+<h4>
+
+<div class="smaller">
```

Comment:
@nimmolo: How do I make the font size for the rest of the page -- including the footer -- the same as the font size on all our other pages?
What I did here is unsatisfactory.

Reply:
Hmm. IIRC many panels on other pages use a class like "name-text" to make the text smaller, and the footers you're thinking of probably have a class like that on them too. 

It could also be there's a text size set on `panel` itself, but i don't think so.

Reply:
Everything on the page -- not just what's included in `app/views/controllers/projects/violations/index.html.erb` was too big. (That's why I made the div `smaller`.) For example, the Edit Translations footer  is too large.
<img width="635" alt="Screenshot 2024-03-14 at 7 41 38 AM" src="https://github.com/MushroomObserver/mushroom-observer/assets/45460/b27c2679-27d2-40a7-ab8d-b0ca29a0b479">

I'll try to figure out why that happens.

Reply:
Problem: I messed up the `<h4>` closing tag -- I omitted the slash. 
Interesting that there wasn't a notice about this in VS Code or the Chrome developer's console.
In any event, it's fixed. So I will be Resolving coversation.

Reply:
I know what you mean. I like using the tag helpers, like `tag.h4`, partly for that reason. There's no linting for erb, but there is for pure Ruby.